packages/python/plotly/plotly/basedatatypes.py

```diff
@@ -3862,18 +3862,17 @@ def _perform_update(plotly_obj, update_obj, overwrite=False):
             # This should be valid even if xaxis2 hasn't been initialized:
             # >>> layout.update(xaxis2={'title': 'xaxis 2'})
             for key in update_obj:
+                # special handling for missing keys that match _subplot_re_match
+                if not key in plotly_obj and isinstance(plotly_obj, BaseLayoutType):
```

Comment:
```suggestion
                if key not in plotly_obj and isinstance(plotly_obj, BaseLayoutType):
```
Just for readability, so I don't have to think whether `not` or `and` has higher precedence 😉